homeassistant/components/teslemetry/config_flow.py

```diff
@@ -79,7 +80,7 @@ async def async_step_reauth(
     ) -> ConfigFlowResult:
         """Handle reauth on failure."""
         self._entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
-        return await self.async_step_reauth_confirm()
+        return await self.async_step_reauth_confirm(entry_data)
```

Comment:
Are you sure that you want to trigger confirm with data pre-populated?
This means that the user does not have the ability to cancel before the old credentials are sent.

If that is on purpose, please add a comment to that effect.

Reply:
This was included by mistake, I don't mind creds being retried as we occasionally have false positive 401s but this doesn't even achieve that result. Ill remove.